bin/rmdir/rmdir.1

```diff
@@ -72,16 +72,19 @@ Be verbose, listing each directory as it is removed.
 .Sh EXIT STATUS
 The
 .Nm
-utility exits with one of the following values:
+utility exits with one of the exit codes as follows:
```

Comment:
I like what we had before better. "as follows" seems weird and in this sentence. The mdoc(7) manual uses the term "exit values" (which seems more likely to be a search term than "codes") when describing the standard macro for generating this section, which itself just leaves it at "exits".

Reply:
That's a good point. Thank you for your feedback! I'll revert that.